Fix english typos and grammar in documentation #4398 (#108834)
Azure/azure-content: SECURITY.md

Instead, please report them to the Microsoft Security Response Center (MSRC) at [https://msrc.microsoft.com/create-report](https://aka.ms/opensource/security/create-report).

If you prefer to submit without logging in, email [secure@microsoft.com](mailto:secure@microsoft.com). If possible, encrypt your message with our PGP key; please download it from the [Microsoft Security Response Center PGP Key page](https://aka.ms/opensource/security/pgpkey).

> You should receive a response within 24 hours. If you still need to, please email us to ensure we received your original message. Additional information can be found at [microsoft.com/msrc](https://aka.ms/opensource/security/msrc). 

Reviewer: ```suggestion
You should receive a response within 24 hours. If you do not, you can email us to ensure we received your original message. Additional information can be found at [microsoft.com/msrc](https://aka.ms/opensource/security/msrc). 
```

This information will help us triage your report more quickly.

> More complete reports can contribute to a higher bounty award if you are reporting for a bug bounty. Please visit our [Microsoft Bug Bounty Program](https://aka.ms/opensource/security/bounty) page for more details about our active programs.

Reviewer: ```suggestion
If you are reporting for a bug bounty, providing more complete reports can contribute to a higher bounty award. Please visit our [Microsoft Bug Bounty Program](https://aka.ms/opensource/security/bounty) page for more details about our active programs.
```
